plugin-tester/src/runner.js

```diff
@@ -1,39 +1,23 @@
 const http = require('request-promise-native')
 const decache = require('decache')
 
-const createRequestBody = (port, pluginConfig) => {
-    return {
-        type: 'query',
-        callbackUrl: `http://localhost:${port}/callback`,
-        configuration: pluginConfig
-    }
-}
-
-const localPlugin = (localPath, query) => () => {
-    const request = {
-        body: query
-    }
-    const response = {
-        status: () => {
-            return {
-                send: (message) => { }
-            }
+const ignoredResponse = {
+    status: () => {
+        return {
+            send: (message) => { }
         }
     }
-    decache(localPath);
-    return require(localPath).plugin()(request, response)
 }
 
-const hostedPlugin = (url, query) => () => {
+module.exports.local = (path, port, configReader) => () => {
```

Comment:
the fix is to only invoke the config reading at the point of executing the render, this ensures we always have the latest files